eng/pipelines/common/evaluate-default-paths.yml

```diff
@@ -47,6 +43,9 @@ jobs:
       - eng/pipelines/coreclr/*
       - eng/pipelines/libraries/*
     - subset: libraries
+      include:
```

Comment:
No need to add these. When no `include` is specified it means it includes everything, but the specified subsets on the `exclude` parameter. Which in this case, from `src/native` it only excludes `src/native/eventpipe/*`

Reply:
OK, does libraries subset include `src/native` implicitly somehow and coreclr / mono don't?

Reply:
Yes, because there is no `include:` paths specified for libraries. This is how the mechanism works:

```
Scenarios:
  1. exclude paths are specified
      Will include all paths except the ones in the exclude list.
  2. include paths are specified
      Will only include paths specified in the list.
  3. exclude + include:
      1st we evaluate changes for all paths except ones in excluded list. If we can not find
      any applicable changes like that, then we evaluate changes for incldued paths
      if any of these two finds changes, then a variable will be set to true.
  In order to consume this variable in a yaml pipeline, reference it via: $[ dependencies.<JobName>.outputs["<StepName>_<subset>.containschange"] ]
  Example:
  -difftarget ''HEAD^1'' -subset coreclr -includepaths ''src/libraries/System.Private.CoreLib/*'' -excludepaths ''src/libraries/*+src/installer/*''
  This example will include ALL path changes except the ones under src/libraries/*!System.Private.CoreLib/*
```

Reply:
Yup, just read your other comments. Thanks for the link.

Reply:
We made this change previously because coreclr and mono builds weren't running when `src/native/eventpipe` changed: https://github.com/dotnet/runtime/commit/565f3ee5ad68dad35305e9157e37fcdccc2aa2b6. Looks like point 3 didn't hold there for coreclr and mono since exclude list doesn't have src/native?

Reply:
Reading this comment (from parent template): https://github.com/dotnet/runtime/blob/c158cb2fe35c17d8fb7de802c1ffba74c2b936b2/eng/pipelines/common/evaluate-paths-job.yml#L16-L24
looks like "all paths" refers to all paths of that subset, not globally?

Reply:
That comment is wrong apparently it meant: 

```
This example will exclude ALL path changes under src/libraries/ except the ones under src/libraries/*!System.Private.CoreLib/* 
```

`include -> exclude`.

Reply:
> Looks like point 3 didn't hold there for coreclr and mono since exclude list doesn't have src/native?

Yeah, weird. Now I'm curious why. Let me run a test real quick.